doc/content/getting-started/migrating/gateway-migration.md

```diff
@@ -6,6 +6,8 @@ aliases: "/getting-started/migrating-from-v2/gateway-migration"
 
 Migrating gateways to {{% tts %}} is an easy, two step process.
 
+{{< warning >}} Migrating **The Things Indoor Gateway (TTIG)** from {{% ttnv2 %}} to {{% tts %}} is currently not achievable, but is expected to be by the end of Q2 2021. {{</ warning >}}
```

Comment:
I would say that it is work in progress, `not achievable` sounds to me as if we're going to solve something impossible by the end of Q2 :joy: